### PR TITLE
feat: add python runtime and plugin

### PR DESCRIPTION
**Pull request by wesbillman**

### CODE REVIEW

File: internal/buildengine/stubs.go [hunk 1]
```diff
@@ -103,7 +103,7 @@ func generateStubsForEachLanguage(ctx context.Context, projectRoot string, modul
 	}
 	err := wg.Wait()
 	if err != nil {
-		return fmt.Errorf("failed to generate go stubs: %w", err)
+		return fmt.Errorf("failed to generate language stubs: %w", err)
```
Comment (matt2e) [inline internal/buildengine/stubs.go:106]: 🫡

**Review by matt2e — APPROVED** [review]

**Review by mistermoe — APPROVED** [review]

File: python-runtime/cmd/ftl-language-python.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+	svc := pythonplugin.New()
+	logger.Infof("ftl-language-python starting to serve on %v", cli.Bind)
+	err := rpc.Serve(ctx,
+		cli.Bind,
+		rpc.GRPC(languagepbconnect.NewLanguageServiceHandler, svc),
+		rpc.PProf(),
+	)
+	logger.Errorf(err, "ftl-language-python stopped serving")
+	kctx.FatalIfErrorf(err)
```
Comment (alecthomas) [inline python-runtime/cmd/ftl-language-python.go:57]: This should be using `internal/plugin.Start()`, which handles all this for you automatically.

**Review by alecthomas — APPROVED** [review]
We'll also need to add Python to the ftl-runner Docker image.

File: python-runtime/cmd/ftl-language-python.go (hunk 2 above)
Comment (wesbillman) [reply python-runtime/cmd/ftl-language-python.go:57]: Awesome! I'll check with @matt2e too and we can update the goplugin branch as well. Thanks!

Comment (wesbillman) [reply python-runtime/cmd/ftl-language-python.go:57]: https://github.com/TBD54566975/ftl/pull/3193

**Pull request merged**